Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -1322,9 +1323,14 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
                     }
                 }
                 
+                if !LinkResolutionMigrationConfiguration.shouldUseHierarchyBasedLinkResolver {
```

Comment:
Should this be merged with the above if block?

Reply:
Yes. Thanks for noticing.